operator/cmd/mesh/upgrade.go

```diff
@@ -273,7 +273,7 @@ func platformBasedTar() (tarExtension string) {
 	case "linux":
 		tarExtension = runtime.GOOS + "-" + runtime.GOARCH + ".tar.gz"
 	case "windows":
-		tarExtension = runtime.GOOS + ".zip"
+		tarExtension = "win.zip"
```

Comment:
How do we the `fetchUrl` in the original issue, can we share the same constant?

Reply:
Its in istio/release-builder, so not easy to share